ui/components/Patterns/PatternConfigurator.js

```diff
@@ -0,0 +1,302 @@
+import React, { useState, useEffect } from "react";
+import {
+  AppBar,
+  FormControl,
+  IconButton,
+  MenuItem,
+  TextField,
+  Toolbar,
+  Tooltip,
+  Autocomplete,
+  Select,
+} from "@mui/material";
+import { styled } from "@mui/system";
+import DeleteIcon from "@mui/icons-material/Delete";
+import FileCopyIcon from "@mui/icons-material/FileCopy";
+import ListAltIcon from "@mui/icons-material/ListAlt";
+import SaveIcon from "@mui/icons-material/Save";
+import {
+  getHumanReadablePatternServiceName,
+  getPatternServiceName,
+  isEmptyObj,
+  NameToIcon,
+  getMeshProperties,
+} from "./helpers";
+
+const PattrenBar = styled(AppBar)(({theme}) => ({
+  marginBottom: theme.spacing(2),
+  backgroundColor: theme.palette(white),
```

Comment:
@subhradwip01  there is an error here , its a syntax error, change  it to `theme.palette.white`

Reply:
Ok got it...